webapp/src/components/dashboard/products/DeleteProduct.tsx

```diff
@@ -95,127 +94,135 @@ export default function DeleteProduct(props: DeleteProductProps): JSX.Element {
   const openDialog = () => {
     setDialogOpen(dialogOpen + 1);
   };
-  return (
-    <React.Fragment>
-      <Container component="main" maxWidth="md" sx={{ mb: 4 }}>
-        <Paper
-          variant="outlined"
-          sx={{ my: { xs: 3, md: 6 }, p: { xs: 2, md: 3 } }}
-        >
-          <h1 style={{ margin: 8 }}>Delete a product</h1>
-
-          <TextField
-            id="outlined-select-currency"
-            select
-            label="Select"
-            fullWidth
-            style={{ margin: 8 }}
-            onChange={handleChange}
-          >
-            {products.map((product) => (
-              <MenuItem key={product.code} value={product.code}>
-                {product.name + " (" + product.description + ")"}
-              </MenuItem>
-            ))}
-          </TextField>
-
-          <Stack
-            direction="row"
-            spacing={2}
-            justifyContent="space-evenly"
-            alignItems="stretch"
+
+  if (
```

Comment:
Probably these lines could be refactored into a parent component where the children is UpdateProduct and DeleteProduct and you pass it as a chidlren.

Reply:
I really don't get your point here. What do you mean?